assembl/lib/sqla.py

```diff
@@ -1690,12 +1691,14 @@ def orm_update_listener(mapper, connection, target):
         return
     session = object_session(target)
     if session.is_modified(target, include_collections=False):
+        reindex_content(target, 'update')
```

Comment:
OK I added your comment in https://github.com/assembl/assembl/commit/f0be89cae3277fbe51364cbc59cd6b0295f26943